### PR TITLE
Enhance 'show' command

### PR DESCRIPTION
**Pull request by jleveque**
- Add 'show interfaces sfp' subcommand to wrap sfputil
 - Remove 'sfp' arg from 'show interfaces' command
 - Add '-p/--period' argument and '-c/--clear' flag to 'show interfaces
   counters'
 - Add missing docstrings for Click
- Add sudo to all 'vtysh', 'docker', 'lldpctl', 'lldpshow' and 'sensors' calls

### CODE REVIEW

File: show/main.py [hunk 1]
```diff
@@ -156,23 +150,46 @@ def summary(interfacename, sfp):
 
 # 'counters' subcommand
 @interfaces.command()
-def counters():
+@click.option('-p', '--period')
+@click.option('-c', '--clear', is_flag=True)
+def counters(period, clear):
```
Comment (sihuihan88) [inline show/main.py]: Can you also provide a -a --printall option to print all the information?

Comment (jleveque) [inline show/main.py]: Good catch, thanks! I meant to do that initially, but forgot.

Comment (sihuihan88) [inline show/main.py]: Shall we also provide -a as an argument option as -p and -c such that we don't have to print all the info by default?

Comment (jleveque) [inline show/main.py]: Done.

**Pull request merged**